src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practice-setup.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Set up OpenTelemetry and New Relic: A best practice guide'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: A best practice guide for setting up New Relic monitoring for your application using OpenTelemetry.
+---
+
+This guide will give you some tips and best practices for using OpenTelemetry to instrument your application or service, and then reporting that data to your New Relic account. 
+
+This can be a helpful guide for: 
+
+* Existing New Relic customers who currently use our APM solutions, or other third-party monitoring solutions, but want to switch to OpenTelemetry
+* Any organization that wants to switch over to using OpenTelemetry and New Relic 
+
+This guide will walk you through ten important steps required for getting your OpenTelemetry data working with New Relic.
+
+
+## 1. Instrument applications with OpenTelemetry [#instrument-apps]
+
+The first step is to instrument application code using OpenTelemetry. For some languages, such as Java, .NET, Python, and Node.js, OpenTelemetry provides an auto-instrumentation approach. This is a great starting point for many applications that can be built on later. Instructions for auto-instrumentation can be found here: 
+
+* [Java auto-instrumentation](https://opentelemetry.io/docs/instrumentation/java/automatic) 
+* [.NET auto-instrumentation](https://opentelemetry.io/docs/instrumentation/net/automatic) 
+* [Node.js auto-instrumentation](https://opentelemetry.io/docs/instrumentation/js/getting-started/nodejs)
+* [Python auto-instrumentation](https://opentelemetry.io/docs/instrumentation/python/automatic)
+
+Manual instrumentation can be used for languages that don't support auto-instrumentation, such as [Go](https://opentelemetry.io/docs/instrumentation/go/manual). It can also be used to augment telemetry collected via auto-instrumentation. 
+
+Auto-instrumentation collects traces, and in some cases, metrics. Both of these signals are important for observing your applications. 
+
+OpenTelemetry can also be used to capture logs. To allow us to correlate OpenTelemetry traces with our [logs-in-context feature](/docs/logs/logs-context/logs-in-context), ensure the `service.name` attribute is included with log entries. For more about reporting OpenTelemetry logs, see our [OpenTelemetry logs docs](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs). 
+
+## 2. Deploy the OpenTelemetry Collector [#deploy-collector]
+
+Next, we'll want to deploy the OpenTelemetry Collector, which is recommended for using OpenTelemetry in production as it offloads tasks such as batching, retries, and encryption. 
```

Comment:
@zuluecho9 there's a lot of information in this collector section. Could you segment it by inserting some h3s or maybe make a list of key collector points.

h3 Collector setup options (discuss the agent vs. gateway)

h3 Collector configuration tips

* As part of the collector configuration, it's important to use the [cumulative-to-delta processor](https://docswebsitedevelop-newotelbestpracticegetstarted.gatsbyjs.io/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics/), which converts metrics with cumulative temporality to a delta format.
* To avoid data being dropped, we also recommend [truncating](https://docswebsitedevelop-newotelbestpracticegetstarted.gatsbyjs.io/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-attributes/) any attributes greater than 4095 characters in length. For an example of applying these practices, see [this collector config](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/collector).
* To collect infrastructure metrics, include the [host metrics receiver](https://docswebsitedevelop-newotelbestpracticegetstarted.gatsbyjs.io/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts/) in the collector config. When using the host receiver as part of the collector configuration, New Relic automatically detects host metrics as part of a host entity and will synthesize its golden metrics (providing the same experience as that resulting from using our infrastructure agent).
* If you have large OpenTelemetry deployments, you will also need to choose a strategy for scaling the collector, both for performance and resiliency benefits. For details on scaling the collector, see [the OpenTelemetry docs](https://opentelemetry.io/docs/collector/scaling).
* If your OpenTelemetry-instrumented services are running in a Kubernetes environment, follow the steps in [Link OpenTelemetry-instrumented applications to Kubernetes](https://docswebsitedevelop-newotelbestpracticegetstarted.gatsbyjs.io/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes/). This ensures that Kubernetes data in New Relic is correlated with OpenTelemetry data.